src/modules/fingerprint_acl/fp_acl.h

```diff
@@ -44,10 +45,22 @@
 
 typedef char username[FP_ACL_USERNAME_MAX_LENGTH];
 
+struct unabto_fcm_token {
```

Comment:
if it is just a null terminated string like the username define it as typedef char unabto_fcm_token[FCM_TOKEN_LENGTH];